docs/querying/sql-functions.md

```diff
@@ -1426,11 +1444,29 @@ If you do not supply an `offset`, returns the value evaluated at the row followi
 
 ## LEAST
 
-`LEAST([expr1, ...])`
+Returns the minimum value from the provided arguments. The expressions must all be convertible to a common data type. 
```

Comment:
Should this say "expressions" instead of "arguments" for consistency? Also, this [this comment](https://github.com/apache/druid/pull/16930/files#r1742738370) regarding "convertible to a common data type."